packages/blade/src/components/ChatBubble/SelfMessageBubble.web.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import BaseBox from '~components/Box/BaseBox';
+import { AlertCircleIcon } from '~components/Icons';
+import { Text } from '~components/Typography';
+
+const SelfMessageBubble = ({
+  children,
+  isError,
+  errorText = 'Message not sent. Tap to retry.',
+  onClick,
+  messageType,
+}: {
+  children: React.ReactNode | string;
+  isError?: boolean;
+  errorText?: string;
+  onClick?: () => void;
+  messageType: 'last' | 'default';
+}): React.ReactElement => {
+  return (
+    <BaseBox
+      display="flex"
+      gap="10px"
+      flexDirection="column"
+      onClick={onClick}
+      cursor={isError ? 'pointer' : 'default'}
+    >
+      <BaseBox
+        maxWidth="240px"
```

Comment:
is this maxWidth set from design side? Shouldn't this be controlled by the consumer and our chatbubble be fluid to accommodate any UI?

Reply:
Yes, maxWidth is set from the design for now. It should be, but currently, Bubble should only be used in specific UIs

Reply:
I don't think adding maxWidth is a good idea, we did the same thing with Accordion and it backfired causing a lot of issues because: 

Once you add maxWidth removing it in future becomes kind of a styling breakage for consumers. 

It should be fluid and expose maxWidth so consumers can add it.

Reply:
It should still have a default maxWidth and then expose to consumers right? otherwise the messages will overflow till the end of chatbox

Reply:
checked with design, we can remove maxWidth, allow consumers to pass it.

Reply:
> It should still have a default maxWidth and then expose to consumers right?

No. That's what caused the [unintuitive](https://razorpay.slack.com/archives/CMQ3RBHEU/p1738575481446369?thread_ts=1738575045.916759&cid=CMQ3RBHEU) issues that consumers faced. 


> It should still have a default maxWidth and then expose to consumers right? otherwise the messages will overflow till the end of chatbox

<Box width="500px">
  <ChatBubble />
</Box>

ChatBubble should fluidly take it's parent's width.